src/components/skills-quiz/SkillsQuiz.jsx

```diff
@@ -14,7 +14,7 @@ import SkillsQuizV2 from '../skills-quiz-v2/SkillsQuiz';
 const SkillsQuiz = () => {
   const { enterpriseConfig } = useContext(AppContext);
   const PAGE_TITLE = `Skills Quiz - ${enterpriseConfig.name}`;
-  const v1 = false;
+  const v1 = true;
```

Comment:
You could use a feature flag ... e.g. https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/032fab758d58464adc5db46280602f7079631935/src/components/course/data/utils.jsx#L875C45-L875C45